src/libopenrave/kinbodyjoint.cpp

```diff
@@ -469,7 +469,10 @@ void KinBody::JointInfo::DeserializeJSON(const rapidjson::Value& value, dReal fU
     orjson::LoadJsonValueByKey(value, "id", _id);
 
     orjson::LoadJsonValueByKey(value, "parentLinkName", _linkname0);
-    orjson::LoadJsonValueByKey(value, "anchors", _vanchor);
+    if (value.HasMember("anchors")) {
```

Comment:
@Puttichai this way will be more efficient.

```c++
const bool anchorFound = orjson::LoadJsonValueByKey(value, "anchors", _vanchor);
if( anchorFound ) {
    _vanchor *= fUnitScale;
}
```

Reply:
Thanks! I updated loading of both `"anchors"` and `"_tLocalTool"`.